react-framework-master/PostgreServer/dbconnection/connection.js

```diff
@@ -0,0 +1,11 @@
+const Pool = require('pg').Pool;
+const pool = new Pool({
```

Comment:
Might be a good idea to write these credentials to a file that you exclude from a .gitignore file, that way you only have variables in place of raw values on Github

Reply:
okay will do